modules/adding-a-data-connection-to-your-data-science-project.adoc

```diff
@@ -37,8 +38,8 @@ The *Add data connection* dialog opens.
 . Click *Add data connection*.
 
 .Verification
-* The data connection that you added appears in the *Data connections* section on the *Details* page for the project.
-* If you selected a workbench, the data connection is visible in the *Workbenches* section on your data science project page.
+* The data connection that you added appears in the *Data connections* tab for the project.
+* If you selected a workbench, the data connection is visible in the *Workbenches* tab for the project.
```

Comment:
I'm not seeing data connections displayed on the Workbenches tab - I can see connected workbenches displayed under *Connected workbenches* for the data connection on the Data connections tab, though.

Reply:
Another good catch, thank you! Updated now!